components/molecules/Quote.js

```diff
@@ -0,0 +1,46 @@
+import PropTypes from "prop-types";
+
+/**
+ * Quote component
+ */
+export function Quote(props) {
+  return (
+    <div
+      title="Quote"
+      className="border-l-4 text-sm md:text-p pl-4 leading-normal font-body"
+    >
+      {props.html === undefined ? (
+        <div>{props.children}</div>
```

Comment:
divs are the wrong element here. Follow this. Important to make our HTML semantic. Helps with screen readers accessibility ect  https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote